corehq/apps/users/migrations/0001_add_location_permission.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-09 02:05
+from __future__ import unicode_literals
+from corehq.sql_db.operations import HqRunPython
+from corehq.apps.users.management.commands.add_location_permission import update_all_roles
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    update_all_roles()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        HqRunPython(migrate),
```

Comment:
No, I don't think that'd be especially hard.  The main reason I did it like this was so it'd happen automatically to (for instance) swiss, NIC, local envs, and third party instances.